src/mokka/src/integration-test/java/pl/hycom/mokka/emulator/mock/MockConfigurationControllerIT.java

```diff
@@ -0,0 +1,179 @@
+            .andExpect(status().isOk());
+
+        Response response = RestAssured.get("/__admin/mappings/" + uuid);
+        expect().statusCode(404).contentType(ContentType.JSON).response();
+
+        mvc.perform(asyncRequest(delete("/configuration/" + id)).contentType(MediaType.APPLICATION_JSON).with(csrf()))
+            .andExpect(status().isOk());
+
+        Assertions.assertEquals(response.getStatusCode(), HttpStatus.NOT_FOUND.value());
+    }
+
+    @Test
+    @WithMockUser(roles="ADMIN")
+    public void enable_mock() throws Exception {
+
+        addMock();
+
+        mvc.perform(asyncRequest(post("/configuration/"+ id +"/disable")).contentType(MediaType.APPLICATION_JSON).with(csrf()))
+            .andExpect(status().isOk());
+
+        mvc.perform(asyncRequest(post("/configuration/"+ id +"/enable")).contentType(MediaType.APPLICATION_JSON).with(csrf()))
+            .andExpect(status().isOk());
+
+        Response response = RestAssured.get("/__admin/mappings/" + uuid);
+        expect().statusCode(200).contentType(ContentType.JSON).response();
+
+        Assertions.assertEquals(response.jsonPath().get("name"), name);
+        Assertions.assertEquals(response.jsonPath().get("request.url"), slash + path);
+        Assertions.assertEquals(response.jsonPath().get("request.method"), httpMethod);
+
+        mvc.perform(asyncRequest(delete("/configuration/" + id)).contentType(MediaType.APPLICATION_JSON).with(csrf()))
+            .andExpect(status().isOk());
+    }
+
+    @Test
+    public void remove_mock() throws Exception {
+
+        addMock();
+
+        mvc.perform(asyncRequest(delete("/configuration/"+ id)).contentType(MediaType.APPLICATION_JSON).with(csrf()))
+            .andExpect(status().isOk());
+
+        Response response = RestAssured.get("/__admin/mappings" + uuid);
+        expect().statusCode(200).contentType(ContentType.JSON).response();
+
+        Assertions.assertEquals(response.getStatusCode(), HttpStatus.NOT_FOUND.value());
+    }
+
+    private MockHttpServletRequestBuilder asyncRequest(MockHttpServletRequestBuilder builder) {
+        return builder.header("x-requested-with", "XMLHttpRequest");
+    }
+
+    private void addMock() throws Exception {
+        String requestBody = "{\"status\":\"" + status + "\",\"httpMethod\":\"" + httpMethod + "\",\"path\":\"" + path + "\",\"id\":"+id+",\"name\":\"" + name + "\",\"type\":\"string\",\"configurationContent\":{\"string\":{\"value\":\"test\"}}}\n";
+
+        mvc.perform(asyncRequest(put("/configuration")).contentType(MediaType.APPLICATION_JSON).with(csrf())
+            .content(requestBody))
+            .andExpect(status().isOk());
+
+        id = mockConfigurationRepository.findAll().iterator().next().getId().toString();
+
+        Response response = RestAssured.get("/__admin/mappings");
+        uuid =  response.jsonPath().get("mappings.uuid").toString().replace("[", "").replace("]", "");
```

Comment:
As mentioned earlier: not needed. We should be able to get "id" from `put` response.
Then id can be converted to uuid with some helper method